Shared/Samples/Add raster from geopackage/AddRasterFromGeopackageView.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddRasterFromGeopackageView: View {
+    /// A map with light gray basemap and a raster layer.
+    @State private var map = Map(basemapStyle: .arcGISLightGray)
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The current viewpoint of the map view.
+    @State private var viewpoint: Viewpoint?
+    
+    var body: some View {
+        MapView(map: map, viewpoint: viewpoint)
+            .onViewpointChanged(kind: .centerAndScale) { viewpoint = $0 }
+            .errorAlert(presentingError: $error)
+            .task {
+                do {
+                    guard let geopackage = GeoPackage(name: "AuroraCO", bundle: .main) else {
+                        return
+                    }
+                    try await geopackage.load()
+                    let rasterLayers = geopackage.rasters.map(RasterLayer.init(raster:))
```

Comment:
```suggestion
                    guard let geoPackage = GeoPackage(name: "AuroraCO", bundle: .main) else {
                        return
                    }
                    try await geoPackage.load()
                    let rasterLayers = geoPackage.rasters.map(RasterLayer.init(raster:))
```

Does the `geopackage` need to be `geoPackage` to match `GeoPackage`?  I also wonder if the sample name needs to change since "GeoPackage" apears to be the official OGC term: https://www.geopackage.org

Reply:
That's a good suggestion. I changed the capitalization for all occurrences, except the title as we need to keep the title consistent with the common design.